### PR TITLE
Feature/acknowledgments on landing page

### LINKED ISSUE #239: Acknowledgements on landing page
## Use Case
It would be nice if the acknowledgements for the Simularium project were posted on the website.

## Acceptance Criteria
There is a section at the bottom of the landing page listing the current acknowledgements, which pulls that information from ACKNOWLEDGEMENTS.md.

## Details
Blair will update ACKNOWLEDGEMENTS.md shortly.

In the absence of UX guidance, since we want to do this soon, I'd say add another section at the end of the landing page with the gray background (like in the "Connect with us" section), title can be "Acknowledgements" and subtitle "We'd like to thank the following people for their contributions to Simularium"

### PR DESCRIPTION
**Pull request by schoinh**
Problem
=======
Closes #239 

Zeplin mockup: https://app.zeplin.io/project/5ceee2a189a82e03ffe7f8a3/screen/613bf2b0e4ea12262a33bacd

Solution
========
In order to pull the markdown content from ACKNOWLEDGMENTS.md in to the LandingPage React component, I used 2 new dependencies: 
* raw-loader to import the .md file into LandingPage.jsx (using `require`... I couldn't get `import` to work??)
* react-markdown to turn the markdown syntax into html syntax

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* This change requires a documentation update

Screenshots (optional):
-----------------------
image

# Note
This is still undergoing legal review, so it shouldn't be merged til that's done

### CODE REVIEW

File: ACKNOWLEDGMENTS.md [hunk 1]
```diff
@@ -20,4 +20,4 @@ Julian Berro, Jiongyi Tan, Margot Quinlan, Lam Nguyen, Tamara Bidone, Glen Hocky
 Ludovic Autin and Ivan Viola
 
 ## For inspiration and/or comments on manuscripts:
-Rick Horwitz, Julie Theriot
```
Comment (schoinh) [inline ACKNOWLEDGMENTS.md:23]: Not sure why this is shown as a diff, but it's the same. Google says "acknowledgements" is British and "acknowledgments" is American, so I changed the file name and use the American spelling elsewhere too. Let me know if this is an issue for some reason.

File: src/components/LandingPage/index.tsx [hunk 2]
```diff
@@ -9,11 +10,13 @@ import BlankCard from "../BlankCard";
 import Footer from "../Footer";
 import { TUTORIAL_PATHNAME } from "../../routes";
 import { CYTOSIM_URL, READDY_URL } from "../../constants";
+const markdown = require("../../../ACKNOWLEDGMENTS.md");
```
Comment (EricJIsaac) [inline src/components/LandingPage/index.tsx:13]: To double check, acknowledgments.md will be found at the same relative path in the deployed website?

**Review by EricJIsaac — APPROVED** [review]

**Review by toloudis — APPROVED** [review]

File: src/components/LandingPage/index.tsx (hunk 2 above)
Comment (schoinh) [reply src/components/LandingPage/index.tsx:13]: Good question... I think that the webpack loader should import the md file as a module and package it all up in the dist folder, so I think we'll be good, right @meganrm ?

Comment (meganrm) [reply src/components/LandingPage/index.tsx:13]: yeah

**Review by meganrm — APPROVED** [review]

**Comment by schoinh** [comment]
Got approvals from comms/legal today (legal said comms review is sufficient)

**Pull request merged**